vipps-login-api.md

```diff
@@ -465,7 +465,7 @@ You can learn more at the [OIDC Standard](https://openid.net/specs/openid-connec
 
 **Request**
 
-[`GET:/.well-known/openid-configuration`](https://vippsas.github.io/vipps-login-api/#/public/discoverOpenIDConfiguration)
+[`GET:/.well-known/openid-configuration`][login-discoveropenid-endpoint]
```

Comment:
I made definitions for these links so it's easier to change them later without affecting all the lines.